test-app/app/templates/docs/index.md

```diff
@@ -18,3 +18,19 @@ This is an Ember addon that wraps the date picker [flatpickr](http://chmln.githu
 ```bash
 ember install ember-flatpickr
 ```
+
+ember-flatpickr requires `flatpickr` as a peer dependency. Install it with your favorite package manager:
+
+```bash
+npm install -D flatpickr
+pnpm add -D flatpickr
+yarn add -D flatpickr
+```
+
+### Default style
```

Comment:
Should we mention themes or is that mentioned elsewhere?

Reply:
There's a brief section on this page:  https://github.com/RobbieTheWagner/ember-flatpickr/blob/2b148eb050bbce62026f38dcf108ca085bcf49a5/test-app/app/templates/docs/usage.md#themes